test/CppTests/OpenDebug/CrossPlatCpp/LaunchCommand.cs

```diff
@@ -34,12 +35,24 @@
         public string MIMode;
 
         [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
-        public string VisualizerFile;
+        public List<string> VisualizerFile { get; set; }
```

Comment:
>List<string> [](http://example.com/codeflow?start=15&length=12)

You might want to make this an `object` instead so that you can test both the old and new format for this.

Reply:
Oh, I see. Nevermind.